app/assets/javascripts/max-attendances.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+    var fieldsCount,
+        maxFieldsCount = 2,
+        $addLink = $('a.add_nested_fields');
+
+    function toggleAddLink() {
+        $addLink.toggle(fieldsCount <= maxFieldsCount);
+    }
+
+    $(document).on('nested:fieldAdded:attendances', function() {
```

Comment:
Yes, the whole JS file is an adaption from an example in [their wiki](https://github.com/ryanb/nested_form/wiki/How-to:-limit-max-count-of-nested-fields) - so best practice I guess.